src/components/dialog/bl-dialog.ts

```diff
@@ -77,18 +82,31 @@ export default class BlDialog extends LitElement {
     }
   }
 
+  private get hasHtmlDialogSupport() {
+    return !!window.HTMLDialogElement;
+  }
+  private get _hasFooter() {
+    return [...this.childNodes].some(node => node.nodeName === 'BL-BUTTON');
+  }
+
   private resizeHandler() {
     this.changeContentHeight();
     this.toggleFooterSticky();
   }
 
   private toggleDialogHandler() {
     if (this.open) {
-      this.dialog.showModal?.();
+      this.hasHtmlDialogSupport
+        ? this.dialog.showModal?.()
+        : this.dialogPolyfill.style.setProperty('--display', 'flex');
+        
       this.onOpen({ isOpen: true });
       document.body.style.overflow = 'hidden';
     } else {
-      this.dialog.close?.();
+      this.hasHtmlDialogSupport
+        ? this.dialog.close?.()
+        : this.dialogPolyfill.style.setProperty('--display', 'none');
```

Comment:
We should not need to do this. Let's just use CSS:

```css
:host() .dialog-polyfill {
   display: none;
   ...
}

:host([open]) .dialog-polyfill {
  display: flex;
}
```

Reply:
Good idea  🎉